app/jobs/finalize_import_job.rb

```diff
@@ -0,0 +1,37 @@
+class FinalizeImportJob < ActiveJob::Base
+  attr_reader :import
+
+  def perform(import_id)
+    @import = Import.find_by id: import_id
+    return unless import
+
+    broadcast_result
+
+    export_errors
+    post_to_slack
+
+    import.finish
+    broadcast_result
+  end
+
+  private
+
+  def import_result
+    @import_result ||= ImportResult.new(@import)
+  end
+
+  def broadcast_result
+    channel = "import_#{@import.id}"
+    ActionCable.server.broadcast channel, import_result
+  end
+
+  def export_errors
+    ImportErrors.export(@import)
+  end
+
+  def post_to_slack
+    url = Rails.application.routes.url_helpers.import_url(@import)
```

Comment:
Sorry for the delay @jonallured. Theoretically, there should only be a single host per app, so it shouldn't be a huge problem even if we change the global config(which is why I didn't think this is that bad). The only thing that sticks in my mind is that AcitonMailer has its own `default_url_options` and there might be a reason why this interface is designed this way. What I could imagine is that having two separate configs would make a clear distinction between the `host` set by the request and ActionMailer's `host`, so you could safely use a custom host name in ActionMailer without accidentally overriding the host set by the request.

But again, it should just work as long as we use a single, consistent host name. You could refactor when you need to use an url helper in another jobs class.